kokkos.presets.json

```diff
@@ -0,0 +1,64 @@
+{
+    "version"   : 3,
+    "cmakeMinimumRequired" : {
+        "major" : 3,
+        "minor" : 22,
+        "patch" : 0
+    },
+    "configurePresets" : [
+        {
+            "name"          : "default",
+            "binaryDir"     : "${sourceDir}/build-with-${presetName}",
+            "cacheVariables" : {
+                "CMAKE_EXPORT_COMPILE_COMMANDS" : {"type" : "BOOL"  , "value" : "ON"},
+                "CMAKE_BUILD_TYPE"              : {"type" : "STRING", "value" : "Release"},
+                "CMAKE_CXX_STANDARD"            : {"type" : "STRING", "value" : "17"},
+                "CMAKE_CXX_EXTENSIONS"          : {"type" : "BOOL"  , "value" : "OFF"},
+                "BUILD_SHARED_LIBS"             : {"type" : "BOOL"  , "value" : "ON"}
+            }
+        },
+        {
+            "name"        : "GNU-OpenMP",
+            "inherits"    : "default",
+            "cacheVariables" : {
+                "Kokkos_ENABLE_OPENMP" : {"type" : "BOOL"  , "value" : "ON"}
+            }
+        },
+        {
+            "name"        : "GNU-Cuda",
+            "inherits"    : "default",
+            "cacheVariables" : {
+                "Kokkos_ENABLE_CUDA"        : {"type" : "BOOL"  , "value" : "ON"},
+                "Kokkos_ENABLE_CUDA_LAMBDA" : {"type" : "BOOL"  , "value" : "ON"},
```

Comment:
If not actually necessary for this PR I would recommend to drop since the option defaults to ON in 4.0 and there is discussion about deprecating it in 4.1 (see kokkos/kokkos#5964)

Reply:
Ok, removed! Thanks for the pointer ;)